### PR TITLE
(new) Oop practice

### PR DESCRIPTION
**Pull request by valeriaDD**

### CODE REVIEW

File: app/Services/ProductionRequestActivityLogger.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class ProductionRequestActivityLogger extends AbstractRequestActivityLogger{
+    
+    protected function collectRequestData(Request $request): array
+    {
+        return [
+            'ip' => $request->ip(),         // ip request to understand where our website is more popular
+            'date-time' => Carbon::now(),   // date time - to know when our website is accessed the most
+            'method' => $request->getMethod(),                       // Method used in request 
+            'uri' => $request->getRequestUri(),                      // Where the request was initiated 
+            'email' => $request->email,
```
Comment (egorbwork) [inline app/Services/ProductionRequestActivityLogger.php:17]: Production logger should not contain the request data.

File: app/Services/DebugRequestActivityLogger.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class DebugRequestActivityLogger extends AbstractRequestActivityLogger{
+    
+    protected function collectRequestData(Request $request): array
+    {
+        return [
+            'date-time' => Carbon::now(), 
+            'method' => $request->getMethod(),                       // Method used in request 
+            'uri' => $request->getRequestUri(),                      // Where the request was initiated 
+            'executionTime' => microtime(true) - LARAVEL_START,      // Total execution time
+            'email' => $request->email,
```
Comment (egorbwork) [inline app/Services/DebugRequestActivityLogger.php:17]: If you want to keep request data in the debug logger, you should keep everything under single key.

**Review by egorbwork — CHANGES_REQUESTED** [review]
I can't approve for the moments loggers.

File: app/Services/DebugRequestActivityLogger.php (hunk 2 above)
Comment (valeriaDD) [reply app/Services/DebugRequestActivityLogger.php]: Changed - request data is collected under the same key and other info about request is collected

File: app/Services/ProductionRequestActivityLogger.php (hunk 1 above)
Comment (valeriaDD) [reply app/Services/ProductionRequestActivityLogger.php]: Changed - request data is erased and collected the fingerprint instead

**Review by egorbwork — APPROVED** [review]
Now looks acceptable.

**Pull request merged**